packages/design-system-docs/package.json

```diff
@@ -9,7 +9,7 @@
   "repository": "CMSgov/design-system",
   "license": "SEE LICENSE IN LICENSE.md",
   "dependencies": {
-    "@cmsgov/design-system": "2.0.0-beta.5",
+    "@cmsgov/design-system": "^2.0.0",
```

Comment:
Wondering why the `^` was used here and not with other dependencies?

Reply:
I think we don't want an exact version, this way when we update `design-system` we dont need to also update this dep. Not 100% sure though, we can adjust in the future if needed